src/Components.tsx

```diff
@@ -1,7 +1,44 @@
 import {useCreateComponent} from './useCreateComponent';
 import {useUpdateWithSetter} from './utils/useUpdateWithSetter';
 import {CollectionOptions, FetchEphemeralKeyFunction} from './types';
-import {NotificationCount} from '@stripe/connect-js';
+import {NotificationCount, InstallState} from '@stripe/connect-js';
+
+export const ConnectAppInstall = ({
+  app,
+  onAppInstallStateFetch,
+  onAppInstallStateChange,
+}: {
+  app: string;
+  onAppInstallStateFetch?: (({appId, state}: InstallState) => void) | undefined;
```

Comment:
I don't think you need the `| undefined` here, since the property is already optional